test/integration/examples/http/job/httpjob_test.go

```diff
@@ -98,6 +99,48 @@ func TestHttp2TlsJob(t *testing.T) {
 
 }
 
+func TestHttp2TcpTlsJob(t *testing.T) {
+
+	//Load CA cert
+	caCert, err := ioutil.ReadFile("/tmp/certs/skupper-service-client/ca.crt")
+	if err != nil {
```

Comment:
Maybe just an assert here to fail the test in case of error?

Reply:
Updated! (16f57f84c6d66e07931a29fdcf14b1f46c6fd0a7)